charts/tekton-apps/Chart.yaml

```diff
@@ -1168,6 +1115,7 @@ description: |
     ```
 
     More complicated example of project containing `wordpress` and `frontend` component.
+    If you need to deploy wordpress component in a namespace different from ArgoCD project's one (i.e. `wordpress`), you need to add `extraDestinationNamespaces: ["wordpress"]` and `argocd.namespace=wordpress`, like in the example below
```

Comment:
Please add more docs that we can also define other components in another namespaces (not only wordpress). So  you can say it like this:

```
Component namespace can be set in 2 ways:
    1. From `project.argocd.namespace` - this is useful when all project components share the same project namespace, i.e. `aaoa-ari`. Example: ...
    2. From `component.argocd.destionationNamespaces` - this can be useful when component has different namespace from the project. For example: ... 
```

But pls locate this docs in common section of this file (not in wordpress related stuff)

Reply:
Added https://github.com/saritasa-nest/saritasa-devops-helm-charts/commit/6c920be0fc27263e5a29bc9e6d8a029ef20a9048